### DISCUSSION TITLE
General Talking

### DISCUSSION THREAD

**Discussion by Bengeljo** [hmes98318/r6-discord-bot#5] [General]
Hey hmes98318,
I would like to know how you grab the information from trn. I would love to add slash commands to develop the bot and my skills. 
In that turn I would probably want to make the tracker bigger, like you can also use it to grab valorant stats or fortnite stats... or any other stat that is available on the site.
We could talk over Discord and you could show it to me directly or maybe you can link a good  video that explains it. 
Hope you are doing good ^^ 
Best regards, 
Bengeljo

**Comment by hmes98318** [comment]
Hi Bengeljo,
It's a great idea. 😊 I can tell you the things I know and if you want to know how to grab the information from trn, 
feel free to add my discord friend (waifu#7414) , I can show you how it works.

Looking forward to your reply
hmes98318
